src/Symfony/Bundle/FrameworkBundle/Client.php

```diff
@@ -104,7 +126,9 @@ protected function doRequest($request)
         if ($this->profiler) {
             $this->profiler = false;
 
-            $this->kernel->boot();
+            if ($this->reboot) {
```

Comment:
this should be removed. We must ensure that the kernel is always booted here, otherwise the call on next line will trigger a fatal error.

``->boot()`` is a no-op in case the kernel is already booted: https://github.com/symfony/symfony/blob/043b8fe955e57928438055fd9bd00fe338573ce8/src/Symfony/Component/HttpKernel/Kernel.php#L121-125